spec/controllers/idv/socure/document_capture_controller_spec.rb

```diff
@@ -355,6 +355,7 @@
     end
 
     it 'returns FOUND (302) and redirects to SSN' do
+      expect(subject.idv_session.doc_auth_vendor).to_not be_nil
```

Comment:
Can you add similar checks to `idv/document_capture_controller_spec.rb` and `idv/link_sent_controller_spec.rb`?

Reply:
Hmmm, I didn't include this check in the other specs since I thought the behavior was sufficiently covered in [spec/policies/idv/flow_policy_spec.rb](spec/policies/idv/flow_policy_spec.rb). Thoughts?

Reply:
I think the flow policy spec updates do a good job of verifying that other steps don't mess with this value inappropriately. But I think what we want to test here is "after the user does the thing that we expect will set doc_auth_vendor, it is actually set". 

extract_pii_from_doc is in the DocumentCaptureConcern, so you could add a test for that method, but I think that it makes sense to add a small assertion to each of the three paths (doc capture, link sent, and verify info for IPP) that the doc auth vendor is set as expected.

Reply:
Got it, should be fairly straightforward to add those checks.

Reply:
Added!